blocklylib-core/src/main/java/com/google/blockly/android/ui/Rotation.java

```diff
@@ -30,19 +30,19 @@
 public final class Rotation {
 
     /** Child is not rotated. */
-    public static final int NONE = 0;
+    public static final int NONE = 1 << 1;
 
     /** Child is always rotated clockwise, so its top is on the right. */
-    public static final int CLOCKWISE = 1;
+    public static final int CLOCKWISE = 1 << 2;
 
     /** Child is always rotated counter-clockwise, so its top is on the left. */
-    public static final int COUNTER_CLOCKWISE = 2;
+    public static final int COUNTER_CLOCKWISE = 1 << 3;
```

Comment:
The numbers behind the flags has changed.
I'm not sure if you think that'll be an issue or not.

Reply:
Non issue IMO, but understand the rationale and happy to accept it.